src/ontology/uberon.Makefile

```diff
@@ -632,6 +632,21 @@ $(TMPDIR)/uberon-edit-plus-tax-equivs.owl: $(OWLSRC) $(TMPDIR)/external-disjoint
 	$(OWLTOOLS_CAT_DYNAMIC) $< $(TMPDIR)/external-disjoints.owl `ls $(BRIDGEDIR)/uberon-bridge-to-*.owl | grep -v emap.owl` --merge-support-ontologies -o -f ofn $@
 .PRECIOUS: $(TMPDIR)/uberon-edit-plus-tax-equivs.owl
 
+# uberon bridges to mba and dmba are now manually curated and generated in https://github.com/obophenotype/ABA_Uberon/tree/new_bridge
+# Note: the bridges are generated in new_bridge branch - this might change in the future, if it breaks here, please check ABA_Uberon repo to make sure that the new bridges are appropriately linked
+
+UBERON_BRIDGE_MBA = "https://raw.githubusercontent.com/obophenotype/ABA_Uberon/new_bridge/src/ontology/new-bridges/new-uberon-bridge-to-mba.owl"
+$(BRIDGEDIR)/uberon-bridge-to-mba.owl: $(SRC)
+	$(ROBOT) annotate -I $(UBERON_BRIDGE_MBA) --ontology-iri $(ONTBASE)/$@ -o $@
+$(BRIDGEDIR)/uberon-bridge-to-mba.obo: $(BRIDGEDIR)/uberon-bridge-to-mba.owl
+	$(ROBOT) convert --input $(BRIDGEDIR)/uberon-bridge-to-mba.owl --output $(BRIDGEDIR)/uberon-bridge-to-mba.obo
+
+UBERON_BRIDGE_DMBA = "https://raw.githubusercontent.com/obophenotype/ABA_Uberon/new_bridge/src/ontology/new-bridges/new-uberon-bridge-to-dmba.owl"
+$(BRIDGEDIR)/uberon-bridge-to-dmba.owl: $(SRC)
+	$(ROBOT) annotate -I $(UBERON_BRIDGE_DMBA) --ontology-iri $(ONTBASE)/$@ -o $@
+$(BRIDGEDIR)/uberon-bridge-to-dmba.obo: $(BRIDGEDIR)/uberon-bridge-to-dmba.owl
+	$(ROBOT) convert --input $(BRIDGEDIR)/uberon-bridge-to-dmba.owl --output $(BRIDGEDIR)/uberon-bridge-to-dmba.obo
```

Comment:
General tip, insread of spelling out the --output parameter, use the much shorter `-o $@`. No need now, just for you to know.

Reply:
@matentzn Disagree: Short options are nice on the command line, but in scripts and Makefiles, I think long, explicit options should be preferred.

Reply:
I agree with that in general, I am lobbying for that. but not use $@ introduces unnecessary redundancy in Makefiles. use then `--output $@` at least?

Reply:
Sorry I misunderstood your comment, I thought it was solely about `-o` versus `--option`. Yes, I agree about using the target variable (`$@`) wherever possible.